content/300-guides/050-database/880-troubleshooting-orm/100-help-articles/425-nextjs-prisma-client-monorepo.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Using Prisma Client in a Next.js project in a monorepo setup'
+metaTitle: 'Using Prisma Client in a Next.js project in a monorepo setup'
+metaDescription: 'Using Prisma Client in a Next.js project in a monorepo setup'
+---
+
+## Problem
+
+If you use Prisma Client in a [Next.js](https://nextjs.org/) application within a monorepo, you will likely run into an error such as the following:
+
+```
+Error: ENOENT: no such file or directory, open '/prisma-next-mono-reproduction/packages/service/.next/cache/webpack/client-development/schema.prisma'
+```
+
+Assume you have a monorepo with the following structure:
+
+```
+.
+├── packages/
+│   ├── prisma/
+│   │   ├── client/ <-- Custom output location for the generated Prisma Client
+│   │   └── schema.prisma
+│   └── service/
+│       ├── pages/
+│       │   └── api/
+│       │       └── test.js
+│       ├── next.config.js
+│       └── package.json
+├── pnpm-workspaces.yaml
+├── package.json
+└── vercel.json
+```
+
+This error mentioned above occurs as a result of a bundling problem during Next.js's bundling process. The `schema.prisma` file is treated as an artifact that should be bundled along with the rest of the `/api/test` endpoint's bundle.
```

Comment:
The problem is not the `schema.prisma` that is part of the file tree shown above.
The problem is a copy of it that is included in the generated Prisma Client.
(Same is true for the Query Engine - it is not the one in `node_modules` being missing, but the copy in the generated Client.)